Awesome Books Modules setup

inakiesk opened the pull request:
This branch is dedicated to using modules to organize the website while maintaining its functionalities.
The features are the following:
-Used ES6 syntax.
-Recreated the Awesome books app using modules.
-Kept all functionalities.
-Refactored functions with arrow method.
-Used Luxon to add a date and time function that displays the correct date and time on the website and imported it as a module (the values are updated every 3 seconds).

inakiesk committed:
Remove localCheck function from index.js file and add it as a module

inakiesk committed:
Fix localCheck module

inakiesk committed:
Modify main display function to a class and add as a module, add navbar as a module

inakiesk committed:
Add date and time function and add it as a module

inakiesk committed:
Fix linter errors

zieeco reviewed (changes requested):
Hi @inakiesk,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments below._

- [ ] Please, kindly add the licensed content in your MIT as this should be a link that shows up the licensed content when clicked. do fix this!
- [ ] Kindly also, add the link to the issues page of this project in the "Contributing" section to aid user in going over the issues page and to make your readme more professional.


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

inakiesk committed:
Add MIT license file and fix Readme

inakiesk commented:
New changes:
-Added the MIT license file and put the link to it inside the README.
-Added the link to the issues page in the contributing section of README.

inakiesk committed:
Fix typo

codepantha reviewed (approved):
# STATUS: APPROVED 🟢 

## Hi @inakiesk 👋, 

You have done well implementing all the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/5oGIdt1xapQ76/giphy.gif"/>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
